does not compile (within ocrd_all, Ubuntu 18.04)

```
jb@pers16:~/ocrd1804/ocrd_all> git log|head 
commit 0f7d5023992d19faa497096d445d88571fa1dbfe
Merge: 84bc669 f35fd3b
Author: Stefan Weil <sw@weilnetz.de>
Date:   Tue Feb 11 15:56:51 2020 +0100

    Merge pull request #61 from OCR-D/update-2020-02-11
    
    update 2020-02-11
```

```
jb@pers16:~/ocrd1804/ocrd_all/ocrd_olena> git log|head 
commit e0d8858fa24ca63bb1047045cc9ed05fdbb256fb
Merge: 55e8042 6e76bfe
Author: Konstantin Baierer <kba@users.noreply.github.com>
Date:   Thu Feb 6 14:56:43 2020 +0100

    Merge pull request #27 from stweil/work
    
    Add new macro BUILD_DIR for build directory

commit 6e76bfe3123996f4df0f13c3f2db1067dda280b0
```

`make all` within ~/ocrd1804/ocrd_all raises this error:

```
make[6]: Verzeichnis „/home/jb/ocrd1804/ocrd_all/venv/build/ocrd_olena/scribo/demo/xml2doc“ wird betreten
/usr/bin/moc -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_XML_LIB -DQT_CORE_LIB -I. -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtXml -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -isystem /usr/include/libdrm -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ /home/jb/ocrd1804/ocrd_all/ocrd_olena/repo/olena/scribo/demo/xml2doc/common.hh -o common.moc.cc
Unknown options: isystem, isystem, isystem, isystem, isystem.
```

```
rm -r ocrd_all
git clone ...
cd ocrd_all
sudo make deps-ubuntu
make all
...
...
make[6]: Verzeichnis „/home/jb/ocrd1804/ocrd_all/venv/build/ocrd_olena/scribo/demo/xml2doc“ wird betreten
/usr/bin/moc -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_XML_LIB -DQT_CORE_LIB -I. -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtXml -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -isystem /usr/include/libdrm -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++  /home/jb/ocrd1804/ocrd_all/ocrd_olena/repo/olena/scribo/demo/xml2doc/common.hh -o common.moc.cc
Unknown options: isystem, isystem, isystem, isystem, isystem.
Makefile:1140: recipe for target 'common.moc.cc' failed
```

--- a/Makefile
+++ b/Makefile
@@ -90,6 +90,7 @@ $(BUILD_DIR)/config.status: $(OLENA_DIR)/configure
 			--prefix=$(PREFIX) \
 			--disable-doc \
 			--disable-dependency-tracking \
+			--with-qt=no \
 			--with-tesseract=no \
 			--enable-scribo SCRIBO_CXXFLAGS="-DNDEBUG -DSCRIBO_NDEBUG -O2"
 

